includes/class-wcpdf-main.php

```diff
@@ -588,6 +590,50 @@ public function init_tmp () {
 		}
 	}
 
+	public function load_custom_and_local_fonts() {
+		require WPO_WCPDF()->plugin_path() . '/vendor/autoload.php';
+
+		$options = array(
+			'defaultFont'				=> 'dejavu sans',
+			'tempDir'					=> $this->get_tmp_path( 'dompdf' ),
+			'logOutputFile'				=> $this->get_tmp_path( 'dompdf' ) . "/log.htm",
+			'fontDir'					=> $this->get_custom_fonts_path(),
+			'fontCache'					=> $this->get_custom_fonts_path(),
```

Comment:
I think the above two should be `$this->get_tmp_path( 'fonts' )`? After all that's what we want to write to!

Reply:
And our custom fonts dir?

Reply:
that's the _source_ we're using (fonts we want to merge into the local fonts)

Reply:
anyway, that's what I originally proposed ([here](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/issues/230#issuecomment-972835273)), maybe you had a different tactic in mind, but in that case please describe the process first. Maybe that's a good idea regardless of the method you use: make a high level walkthrough of what your PR is supposed to do.

Reply:
I was doing 2 separated merges, first merging the custom fonts from the plugin and then from the local, but this way seems cleaner.